csi/exporter.py

```diff
@@ -41,10 +41,9 @@ def metrics():
     }
 
     if os.environ.get("CSI_ROLE", "-") == "nodeplugin":
-        pod_name_path = '/etc/hostname'
-        with open(pod_name_path, 'r') as pod_fd:
-            pod_name = pod_fd.read().strip()
-            data["pod"].update({"pod_name": pod_name})
+        # pool & pvc details can be read from provisioner
```

Comment:
Is Pool & PVC details coming out of nodeplugin? I thought only pod details such as name, memory, cpu usage is coming out of it.
If this commented on, then even prometheus related data of nodeplugin must be commented out.

Reply:
- yep but they are not being utilised by operator so it was just adding overhead